cmd/unmanageddetector/main.go

```diff
@@ -73,14 +73,16 @@ func main() {
 		logging.Fatal(err, "error getting config to talk to API server")
 	}
 
+	opts := manager.Options{}
+
+	// Disable cache to avoid stale reads (e.g. of pods, which we need do
+	// to determine if a controller pod exists for a namespace).
+	// TODO(jcanseco): Determine if disabling the cache for this controller
+	// is really necessary. Disable it for now to play it safe.
```

Comment:
```suggestion
```

feel free to disregard if it still applies

Reply:
I don't think I can make the determination right away (and not sure I want to risk making a change here)